hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1158,29 +1157,28 @@ object LoadVCF {
     if (line.isFixedCount &&
       (line.getCount == 1 ||
         (isFlag && line.getCount == 0)))
-      (Field(id, baseType, i), (id, attrs), isFlag)
-    else if (baseType.isInstanceOf[TCall])
+      ((id, baseType), (id, attrs), isFlag)
+    else if (baseType.isInstanceOf[PCall])
       fatal("fields in 'call_fields' must have 'Number' equal to 1.")
     else
-      (Field(id, TArray(baseType.setRequired(arrayElementsRequired)), i), (id, attrs), isFlag)
+      ((id, PCanonicalArray(baseType.setRequired(arrayElementsRequired))), (id, attrs), isFlag)
```

Comment:
Why not PField (or what was the original reason to use Field?)

Reply:
Don't need the field here. Passing in the index was just more complicated, and unnecessary.